src/symbol.js

```diff
@@ -24,24 +22,6 @@ Object.keys = function(obj) {
 // implement iterators for IE 11
 const iterator = window.Symbol.iterator;
 
-if (!String.prototype[iterator]) {
```

Comment:
Maybe revert the deletion, but instead change the condition to `!String.prototype[iterator] || !String.prototype.codePointAt`

Reply:
My latest commit adds a miniature polyfill for codePointAt. Your suggestion works too. Let me know which option you prefer.

Reply:
Looks like the polyfilled string iterator is broken for some other reason. I'll try your suggestion

Reply:
@ruphin I am receiving updates and just saw your last comment about polyfilling String iteration with single index access per character. Is it not going to break apps that iterate on Unicode Strings? For sure, if such an app runs on IE11, it must already include a polyfill, but depending on when webcomponentsjs is loading, it may replace the existing polyfill with a "wrong" one and prevent the existing to load since one is already defined.

What do you think about using the polyfill from Mozilla doc?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/codePointAt#Polyfill

It seems to come from https://github.com/mathiasbynens/String.prototype.codePointAt, which include tests.

Reply:
Thanks! I'll look into it

Reply:
I tried making that work, but for unfortunately the `Symbol.iterator` polyfill for String iteration does not work in IE11, even when using a proper `codePointAt` polyfill.

This change is not changing the existing behaviour of webcomponentsjs, it is only fixing some breakage. The old implementation used the exact same polyfill for String iteration, so it won't break existing apps.

If you have an app that depends on iterator behaviour over Unicode Strings, you can load a 'proper' polyfill for these before loading webcomponentsjs.